app/javascript/src/components/editor/DragHandle.svelte

```diff
@@ -28,14 +28,19 @@
   function mouseMove(event) {
     if (!isDragging) return
 
-    const difference = (event[direction] || event.targetTouches[0]?.[direction]) - startPosition
-    currentSize = startSize + difference * (align == "right" ? 1 : -1)
+    requestAnimationFrame(() => {
```

Comment:
Perhaps we can save an indent with a promise. GitHub won't let me do a full suggestion, but that would be somehting like this.

```suggestion
    await new Promise(res => requestAnimationFrame(res))
```
or
```suggestion
    await new Promise(res => setTimeout(res))
```

Reply:
Sure!

Reply:
☝️🤓 Note that, because of the way browsers usually execute the event loop (sync code → async code → request animation frame → timeouts → event handling → repeat [1]), making this a promise may make it run a little bit later than you want.

It probably doesn't affect much here anyways.

[1]: According to MS Copilot, which gave me [this link](https://dev.to/opensign/a-comprehensive-guide-to-the-event-loop-and-asynchronous-programming-in-javascript-46l0) as source. I only remember watching a talk about this from years ago, so I don't know how true it is nowadays.